core/src/main/java/org/apache/calcite/adapter/java/ReflectiveSchema.java

```diff
@@ -235,6 +240,13 @@ private static Enumerable toEnumerable(final Object o) {
         "Cannot convert " + o.getClass() + " into a Enumerable");
   }
 
+  private static @Nullable Double toRowCount(final Object o) {
```

Comment:
`toRowCount` -> `getRowCount` may be better

Reply:
Done in #3634 .